core/src/test/scala/org/apache/spark/DistributedSuite.scala

```diff
@@ -17,6 +17,7 @@
 
 package org.apache.spark
 
+import SparkMasterRegex._
```

Comment:
I'd move this import inside of the `test("local-cluster format")` to avoid complaints about the import ordering. Otherwise, this looks good to me. Thanks!